### DISCUSSION TITLE
OpenAI: could not fetch translation: Connection error.

### DISCUSSION THREAD

**Discussion by cankirca** [WeblateOrg/weblate#11918] [Q&A]
Hello,
 
 Having an issue with enabling OpenAI; here the steps:
 1. Tried via web interface and bash.
 2. Getting Could not fetch translation: Connection error for both.
 3. Enabled debug and loglevel: debug on configuration but there is no more info.
 4. Set model to auto for both.
Btw: There is no strings to translate right now, just completing the setup before adding translations and other components. Also, there is no limit or restriction for the API key on OpenAI Platform interface.

Could you help me to solve that please?

**Comment by nijel** [comment]
What exactly is the error you get? It accesses https://api.openai.com/, maybe it is not reachable for you for some reason?
